src/summary/history/programs/programs-level-two.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import dayjs from "dayjs";
+import SummaryCard from "../../cards/summary-card.component";
+import SummaryCardRow from "../../cards/summary-card-row.component";
+import SummaryCardRowContent from "../../cards/summary-card-row-content.component";
+import { match } from "react-router";
+import { fetchPatientPrograms } from "./programs.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import HorizontalLabelValue from "../../cards/horizontal-label-value.component";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import SummaryCardFooter from "../../cards/summary-card-footer.component";
+import { Trans, useTranslation } from "react-i18next";
+import styles from "./programs-card-level-two.css";
+
+export default function ProgramsLevelTwo(props: ProgramsCardProps) {
+  const [patientPrograms, setPatientPrograms] = React.useState(null);
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    const subscription = fetchPatientPrograms(patientUuid).subscribe(
```

Comment:
I think we should do a check first before calling `fetchPatientPrograms` that checks whether patientUuid is `null` to avoid sending a request with a null `patientUuid` to the server. I have seen this problem occur before.